docs/edgeql/admin/roles.rst

```diff
@@ -19,20 +19,24 @@ Create a role.
 
 .. eql:synopsis::
 
-    CREATE ROLE <name> [ EXTENDING <base> [, ...] ]
+    CREATE [ SUPERUSER ] ROLE <name> [ EXTENDING <base> [, ...] ]
```

Comment:
It's mandatory so make that clear.

```suggestion
    CREATE SUPERUSER ROLE <name> [ EXTENDING <base> [, ...] ]
```

When it becomes optional we can add []